flottform/forms/src/flottform-channel-host.ts

```diff
@@ -0,0 +1,292 @@
+import { toDataURL } from 'qrcode';
+import { Logger } from '../dist';
+import { FlottformState } from '../dist/internal';
```

Comment:
`dist` looks wrong to me?

Reply:
Yes, it should be imported from './internal' 👍🏼